src/Chapter2.hs

```diff
@@ -740,7 +790,9 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains _ [] = []
+contains e l = [x | x <- l, elem e x]
```

Comment:
Wow, great solution 👏🏼  Nice usage of list comprehensions!
You can do it with the empty list as well, so no need to pattern match separately 🙂